src/components/claims/claim-details/components/claim-items/index.tsx

```diff
@@ -1,26 +1,122 @@
-import { Grid } from '@material-ui/core'
+import { Button } from '@material-ui/core'
 import { Contract } from 'api/generated/graphql'
 import { Paper } from 'components/shared/Paper'
+import { useGetClaimItems } from 'graphql/use-get-claim-items'
+import { useGetClaimValuation } from 'graphql/use-get-claim-valuation'
+import { useContractMarketInfo } from 'graphql/use-get-member-contract-market-info'
+import {
+  UpsertClaimItemVariables,
+  useUpsertClaimItem,
+} from 'graphql/use-upsert-claim-item'
+import { Spacing } from 'hedvig-ui/spacing'
 import React from 'react'
-import { ItemForm } from './components/ItemForm'
-import { ItemList } from './components/ItemList'
+import styled from 'react-emotion'
+import { Chips } from './chips/Chips'
+import { TotalValuationChip } from './chips/components/TotalValuationChip'
+import { ItemForm } from './item-form/ItemForm'
+import { ItemList } from './item-list/ItemList'
+import { isEmpty, isValidDate, isValidNumber } from './utils'
+
+const BottomWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const ChipsWrapper = styled.div``
+
+const formIsValid = (request: UpsertClaimItemVariables) => {
+  if (!request) {
+    return false
+  }
+
+  const {
+    itemFamilyId,
+    itemTypeId,
+    purchasePriceAmount = '',
+    dateOfPurchase = '',
+    customValuationAmount = '',
+  } = request
+
+  if (!itemFamilyId || !itemTypeId) {
+    return false
+  }
+
+  if (!isEmpty(purchasePriceAmount) && !isValidNumber(purchasePriceAmount)) {
+    return false
+  }
+
+  if (!isEmpty(dateOfPurchase) && !isValidDate(dateOfPurchase)) {
+    return false
+  }
+
+  return isEmpty(customValuationAmount) || isValidNumber(customValuationAmount)
+}
 
 export const ClaimItems: React.FC<{
   claimId: string
-  memberId: string | null
-  contract?: Contract | null
+  memberId: string
+  contract: Contract | null
 }> = ({ claimId, memberId, contract }) => {
+  const [claimItems] = useGetClaimItems(claimId)
+  const [contractMarketInfo] = useContractMarketInfo(memberId)
+  const { typeOfContract } = { ...contract }
+  const [claimValuation] = useGetClaimValuation(claimId, typeOfContract)
+  const [upsertClaimItem, { loading }] = useUpsertClaimItem(claimId)
+
+  const { preferredCurrency = 'SEK' } = { ...contractMarketInfo }
```

Comment:
No defaulting to 'SEK' pls :)
```suggestion
  const { preferredCurrency } = { ...contractMarketInfo }
```